duckduckgo/Android
# Replace whitelist strings references in English and Spanish

Task/Issue URL: https://app.asana.com/0/414730916066338/1180899597720226
Tech Design URL: 
CC: 

**Description**:
This PR replaces references to "whitelist" in English and Spanish. Until we get new translations for all the languages we have moved the English strings to the untranslated file for clarity.

**Steps to test this PR**:
1. Launch the app.
1. Go to a website.
1. Open the menu and see there are no references to whitelist.
1. Open settings and see there are no references to whitelist.
1. Open "Unprotected sites" and see there are no references to whitelist.
1. Repeat in Spanish with no references to "lista blanca".

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
